shims/spark320/src/main/scala/com/nvidia/spark/rapids/shims/spark320/Spark320Shims.scala

```diff
@@ -805,4 +808,30 @@ class Spark320Shims extends Spark32XShims {
   override def hasAliasQuoteFix: Boolean = true
 
   override def hasCastFloatTimestampUpcast: Boolean = true
+
+  /** Return list of matching predicates present in the plan */
+  override def findOperators(plan: SparkPlan, predicate: SparkPlan => Boolean): Seq[SparkPlan] = {
+    def recurse(
+        plan: SparkPlan,
+        predicate: SparkPlan => Boolean,
+        accum: ListBuffer[SparkPlan]): Seq[SparkPlan] = {
+      plan match {
+        case _ if predicate(plan) =>
+          accum += plan
+          plan.children.flatMap(p => recurse(p, predicate, accum)).headOption
+        case a: AdaptiveSparkPlanExec => recurse(a.executedPlan, predicate, accum)
+        case qs: BroadcastQueryStageExec => recurse(qs.broadcast, predicate, accum)
+        case qs: ShuffleQueryStageExec => recurse(qs.shuffle, predicate, accum)
+        case c: CommandResultExec => recurse(c.commandPhysicalPlan, predicate, accum)
```

Comment:
`CommandResultExec` is new in Spark 3.2

Reply:
For the sake of keeping Shims small can we hack something like this instead of doing a pattern match:
```scala
    plan.productIterator.foreach { planMember =>
        if (planMember.isInstanceOf[SparkPlan]) {
          recurse(planMember.asInstanceOf[SparkPlan], predicate, accum)
        }
      }
```

Reply:
Each of the items called out in the pattern hides part of the rest of the plan in different ways. If it were generic it would not need to be a part of the shim at all.

Reply:
Sure, I can update this

Reply:
Sorry, misread the comments here. Ignore my previous comment,